lib/datasource/pypi.js

```diff
@@ -23,13 +26,15 @@ async function getDependency(purl) {
         dependency.homepage = dep.info.home_page;
       }
     }
-    dependency.versions = {};
+    dependency.releases = [];
     if (dep.releases) {
-      Object.keys(dep.releases).forEach(release => {
-        dependency.versions[release] = {
-          date: (dep.releases[release][0] || {}).upload_time,
-        };
-      });
+      const versions = Object.keys(dep.releases)
+        .filter(version => isVersion(version))
+        .sort(sortVersions);
+      dependency.releases = versions.map(version => ({
+        version: isVersion(version),
```

Comment:
Should we remove the requirement that `isVersion` returns a clean version and instead make it more obvious with `sanitize()` function for each versioning scheme?